src/stdune/queue.mli

```diff
@@ -0,0 +1,29 @@
+type 'a t
```

Comment:
Should this be `= 'a Stdlib.Queue.t` for code that uses stdune outside `dune`?

Reply:
I could have, but it didn't seem necessary and it prevents us from altering the implementation in the future. If anyone has a use for the type equality, they're free to add it later though.

Reply:
sounds good!